security/security-spring/src/main/java/org/cbioportal/security/spring/CancerStudyPermissionEvaluator.java

```diff
@@ -181,6 +182,15 @@ public boolean hasPermission(Authentication authentication, Serializable targetI
                 }
             }
             return true;
+        } else if ("List<SampleListId>".equals(targetType)) {
```

Comment:
I think since we are only supporting access to web service from website, that there should not be any incorrect id's past to the endpoints.  We may want to re-think this if we make the web service programmatically accessible.